buildifier/buildifier.go

```diff
@@ -136,9 +136,7 @@ func main() {
 
 	diff = differ.Find()
 
-	// TODO(bazel-team): Handle "-" as stdin/stdout mode too.
-
-	if len(args) == 0 {
+	if (len(args) == 0) || (len(args) == 1 && args[0] == "-") {
```

Comment:
Yes, I am aware of that. Usually, I put conditions under the brackets.
Do you want to remove it?